environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java

```diff
@@ -887,6 +890,22 @@ public Weld enableDevMode() {
         return property(DEV_MODE_SYSTEM_PROPERTY, true);
     }
 
+    /**
+     * Registers annotations which will be considered as bean defining annotations.
+     *
+     * NOTE - If used along with {@code <trim/>} bean archives and/or with Weld configuration key
+     * {@code org.jboss.weld.bootstrap.vetoTypesWithoutBeanDefiningAnnotation}, these annotations will be ignored.
+     *
+     * @param annotations annotations which will be considered as Bean Defining Annotations.
+     * @return self
+     */
+    public Weld addBeanDefiningAnnotations(Class<? extends Annotation>... annotations) {
```

Comment:
Does it make sense to document this, e.g. in 15.6.2. Implicit bean archive (there's a NOTE in that section listing what are bean defining annotations)?

Reply:
Chapter 15 is about EE while this feature only concerns SE.
But you are right that we might want to mention this, but maybe in `18.4.2. Bootstrapping CDI SE` instead?

Reply:
Of course you are right, apologies for confusion.